firedrake/interpolation.py

```diff
@@ -113,6 +113,13 @@ class Interpolator(object):
         interpolation over.
     :kwarg freeze_expr: Set to True to prevent the expression being
         re-evaluated on each call.
+    :kwarg access: The access descriptor for combining updates to shared dofs.
```

Comment:
Add something like: "Possible values are `WRITE` and `INC`"?

Reply:
Do `MAX` and `MIN` not work?

Reply:
Oh I've found this note in `interpolate`'s docstring
```
    .. note::

       If you use an access descriptor other than ``WRITE``, the
       behaviour of interpolation is changes if interpolating into a
       function space, or an existing function. If the former, then
       the newly allocated function will be initialised with
       appropriate values (e.g. for MIN access, it will be initialised
       with MAX_FLOAT). On the other hand, if you provide a function,
       then it is assumed that its values should take part in the
       reduction (hence using MIN will compute the MIN between the
       existing values and any new values).
```